### DISCUSSION TITLE
Linux version?

### DISCUSSION THREAD

**Discussion by lordofscripts** [deckerst/aves#1039] [Q&A]
I just started using this app on my Android phone. I was wondering if there are plans to publish a Linux version?

**Comment by deckerst** [comment]
There is no such plan. I use Linux too, but making the app work on it would be a lot of additional work.

**Reply by lordofscripts** [reply]
Any suggestion of a linux-based compatible package that would read all tags embedded by Aves? I would like to be able to use Aves on my Android phone but being able to browse those photos with their metadata in Linux without having to do double work.

**Reply by deckerst** [reply]
Some people use [Digikam](https://www.digikam.org/). I'm not a user myself but it seems like good software.

**Reply by deckerst** [reply]
You can also check out:
- #679
